src/google/cloud/ndb/_datastore_api.py

```diff
@@ -34,6 +34,7 @@
 
 EVENTUAL = datastore_pb2.ReadOptions.EVENTUAL
 EVENTUAL_CONSISTENCY = EVENTUAL  # Legacy NDB
+_DEFAULT_TIMEOUT = None
```

Comment:
Making this the default, means that, effectively, we aren't passing a timeout to gRPC, so it will do whatever it does when you don't pass one. (Use a default? Wait forever? Not sure...)

We could also explicitly pass one. It's not entirely clear what Legacy is doing here, since the Datastore/RPC APIs are completely different. Open to suggestions.